core-rust/core-api-server/src/core_api/handlers/status_network_status.rs

```diff
@@ -29,17 +25,12 @@ pub(crate) fn handle_status_network_status_internal(
                 Ok(Box::new(to_api_committed_state_identifier(identifiers)?))
             })
             .transpose()?,
-        current_state_identifier: Box::new(
-            state_manager
-                .store()
-                .get_top_of_ledger_transaction_identifiers()
```

Comment:
_Note: I now use this function more seriously, so I made the impl explicit (and touched the API a bit)_